thunder/torch/__init__.py

```diff
@@ -3590,253 +3576,191 @@ def cross_entropy(
         lambda: f"Deprecated size_average={size_average} and reduce={reduce} is not supported!",
     )
 
+    _cross_entropy_input_checks(a, target, weight, ignore_index, reduction, label_smoothing)
+
+    # channels dimension is either the first one if no batch dim present (i.e. a.shape[0]),
+    # or right next to it (i.e. a.shape[1]).
+    channels_dim = 1 if a.ndim >= 2 else 0
```

Comment:
`channels` is very particular to image data, can we name this differently, like `softmax_dim`, or `reduction_dim`?

Reply:
I think "class" is the word we're looking for per the [PyTorch docs for `cross_entropy`](https://pytorch.org/docs/master/generated/torch.nn.functional.cross_entropy.html).